Azure/communication-ui-library-ios
# [Bugfix] Solve the retain cycle between videoViewManager and Coordinate

## Purpose

* The bug is the local video stream couldn't be displayed after ending a call with local video on
* The bug cause is the videoViewManager didn't get deallocated when composite is dismissed
* The videoViewManager's `didRenderFirstFrame` catch the coordiante's `ZoomableRenderView`
* The `ZoomableRenderView` catch the videoViewManager

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Test end a call with local video on, see if it's possible to see the grey local view when you restart a call
